Dockerfile

```diff
@@ -43,8 +43,7 @@ RUN gem install --no-ri --no-rdoc r10k \
       puppet-lint \
       onceover \
       rest-client \
-      ra10ke \
-    && ln -s /usr/local/bundle/bin/* /usr/local/bin/
```

Comment:
@TJM Are you sure this symlink operation isn't necessary anymore? It was added to ensure that all gem binaries, including ra10ke, are all available in the path. Perhaps we should update the PATH variable too?

Reply:
It is not relevant anymore, because the scripts (bin dir) is not /usr/local/bundle/bin while using the /opt/rh/(blah) Ruby. If you look in /usr/local/bin, you will find a broken symlink named "`*`" which is not taking a lot of space, but also not desirable. The path was already updated to include the "extra" ruby bin dir. 

~tommy